### DISCUSSION TITLE
Persist wallet metadata/contacts in chain

### DISCUSSION THREAD

**Discussion by diomark** [alephium/alephium-frontend#498] [Feature requests]
What about persisting a users wallet metadata in a smart contract on chain? Ie wallet names, contact list, etc. that way when the wallet is loaded on a new device, or same seed is used in multiple devices, they can share settings?

Of course there would be storage rent for this so make it optional?

**Comment by nop33** [comment]
I think it's an interesting idea and would fit very well as a hackathon task! @mvaivre what do you think? I remember we already had this idea ourselves back when discussing the sync between desktop <> mobile wallet.
